.circleci/config.yml

```diff
@@ -116,6 +116,18 @@ references:
       command: |
         echo "Loading wiremock mappings..."
         find spec/wiremock/prison-api/mappings/*.json -exec curl --request POST --url http://localhost:8888/__admin/mappings/ --header 'content-type: application/json' --data-binary "@{}" \;
+  _notify_sentry_release: &notify_sentry_release
+    run:
+      name: Create release and notify Sentry of deploy
+      command: |
+        curl -sL https://sentry.io/get-cli/ | bash
+        export SENTRY_RELEASE=$(sentry-cli releases propose-version)
+        sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
```

Comment:
I'm not convinced `-p $SENTRY_PROJECT` is necessary?

Reply:
> Probably not 👍

Added a reference to the process in the Readme.md

Reply:
> I'm not convinced `-p $SENTRY_PROJECT` is necessary?

For what I could understand from the docs, it explicitly creates a release event associated with the project. It's a bit more involved than sentry detecting a new release is on looking that the release label we defined in Dockerfile

Reply:
Okey doke. I'm easy :)